.travis.yml

```diff
@@ -70,6 +73,27 @@ matrix:
     - python: 3.5
       env:
           - DEPENDS="ipython[notebook] sphinx==1.7.6 matplotlib"
+# Adding ppc64le jobs
```

Comment:
Are all these entries important?

Reply:
Not all entries are important. As there is point of bit concern, hence removed few entries from the PR. Please have a look to it and let me know if required anything else